tests/js/client/shell/transaction/shell-transaction-cluster.js

```diff
@@ -253,35 +275,26 @@ function transactionReplication2ReplicateOperationSuite() {
 function transactionReplicationOnFollowersSuite(dbParams) {
   'use strict';
   const dbn = 'UnitTestsTransactionDatabase';
-  const numberOfShards = 1;
   const cn = 'UnitTestsTransaction';
+  const rc = replicatedLogsHelper.dbservers.length;
+  const isReplication2 = dbParams.replicationVersion === "2";
   let c = null;
-  let isReplication2 = dbParams.replicationVersion === "2";
-
-  return {
-    setUpAll: function () {
-      db._createDatabase(dbn, dbParams);
-      db._useDatabase(dbn);
-    },
-
-    tearDownAll: function () {
-      db._useDatabase("_system");
-      db._dropDatabase(dbn);
-    },
 
-    setUp: function () {
-      db._drop(cn);
-      let rc = Object.keys(replicatedLogsHelper.dbservers).length;
-      c = db._create(cn, {"numberOfShards": numberOfShards, "writeConcern": rc, "replicationFactor": rc});
-    },
+  const {setUpAll, tearDownAll, setUpAnd, tearDownAnd} =
+    replicatedLogsHelper.testHelperFunctions(dbn, {replicationVersion: "2"});
```

Comment:
We probably want to set the replicationVersion based on the dbParams:
```suggestion
    replicatedLogsHelper.testHelperFunctions(dbn, {replicationVersion: isReplication2 ? "2" : "1"});
```

Reply:
Thanks!